### PR TITLE
Reformulate sentence concerning stack allocated types

### PR DESCRIPTION
**Pull request by Hofer-Julian**

### CODE REVIEW

File: _posts/2022-01-14-new-release.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ By using the derive macro, structured data can be handled automatically instead
 
 ### Stack allocated types and reducing heap allocations
 
-Various types that usually used stack-allocated from C were previously heap-allocated in the Rust bindings. With this release, these values are also stack-allocated when used from Rust unless explicitly heap-allocated, e.g. by explicitly placing them into a `Box`.
+Various types that are stack-allocated in C were previously heap-allocated in the Rust bindings. With this release, these values are also stack-allocated when used from Rust unless explicitly heap-allocated, e.g. by explicitly placing them into a `Box`.
```
Comment (sdroege) [inline _posts/2022-01-14-new-release.md:59]: The `usually` here was important :)

Comment (Hofer-Julian) [reply _posts/2022-01-14-new-release.md:59]: If you add usually, then I would read it as: "If it is usually stack allocated in C then it is always stack allocated in Rust unless explicitly heap allocated".

Is that correct?

Comment (sdroege) [reply _posts/2022-01-14-new-release.md:59]: Sounds good to me and better than my version.

Comment (Hofer-Julian) [reply _posts/2022-01-14-new-release.md:59]: Good to know, because I wouldn't know how to determine whether something is "usually" stack allocated :P

Comment (Hofer-Julian) [reply _posts/2022-01-14-new-release.md]: I've added "usually" to the sentence.

**Review by sdroege — APPROVED** [review]

**Pull request merged**